src/assets/images/leoiot-loading/create-loading.sh

```diff
@@ -0,0 +1,2 @@
+rm leoiot-loading.webm
+ffmpeg -framerate 1 -i leoiot-loading-%d.png -vcodec libvpx -acodec libvorbis -auto-alt-ref 0 leoiot-loading.webm
```

Comment:
I am assuming this creates the loading gif. Is this documented somewhere?

Reply:
Would a comment here be enough documentation, or do you want it in the docs repo?

Reply:
it would be better in the docs repo I think. This way you don't need to search in the code for how the animation works.

Reply:
just create a new file under the frontend module and explain what the script does and the logo works or something.

Reply:
https://github.com/leo-iot/leo-iot-docs/commit/9007b092488f8452a267c0815e22915be7dab7af
Should I explain the script more detailed or is this enugh?

Reply:
looks good